domains/_github-pages-challenge-clay990.suraj.json

```diff
@@ -5,5 +5,36 @@
   },
   "record": {
     "TXT": "dee6e1fa6a0e02df5af4edb0e40ac4"
+    "MX": [
+      {
+        "host": "suraj",
+        "value": "mx.zoho.in",
+        "priority": 10
+      },
+      {
+        "host": "suraj",
+        "value": "mx2.zoho.in",
+        "priority": 20
+      },
+      {
+        "host": "suraj",
+        "value": "mx3.zoho.in",
+        "priority": 50
+      }
+    ],
+    "SPF": {
+      "type": "TXT",
+      "host": "suraj",
+      "value": "v=spf1 include:zohomail.in ~all"
+    },
+    "DKIM": {
+      "type": "TXT",
+      "host": "zmail._domainkey.suraj",
+      "value": "v=DKIM1; k=rsa; p=MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQCynLFWf+Or8UXFbj0VF2TV8uwsfgeB8CsNFlW+vh/QxO4D2QGWQsO78Z1zeLA99CvWHMcz028dJoBDWkRiQLjIuagw/HvGnZ6b33eatfB7c9ekMV8pOH4nkCwIlZOgaraQ99ujKVhkbQHjThzSy6XaeqRJydJoLRKJOQUa/fW32QIDAQAB"
+    },
+    "TXT": {
+      "host": "suraj",
+      "value": "dee6e1fa6a0e02df5af4edb0e40ac4"
```

Comment:
Several of these records types are not supported, and the format some of these are placed are incorrect.

Please read the docs https://docs.is-a.dev/ and adjust your pull request accordingly.